app.yaml

```diff
@@ -0,0 +1,6 @@
+runtime: python37
+automatic_scaling:
```

Comment:
Do we need to specify
`liveness_check:
  path: "/status"`
?

Reply:
That's only for app engine flex. It's used by app engine to gauge the health of a node that it doesn't otherwise have any knowledge of how it works. App engine standard is a more controlled environment so they don't need the app itself to provide this.

Reply:
👍 Hooray standard!